requests/CreateProductRequest.go

```diff
@@ -0,0 +1,27 @@
+package requests
+
+import (
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	u "go-fiber-example/m/v2/utils"
+)
+
+type CreateProductRequest struct {
+	Code  string
+	Price string
+}
+
+func ValidateCreateProductRequest(ctx *fiber.Ctx) CreateProductRequest {
```

Comment:
in this function, i think you should probably only do validation according to the name but not parsing you request body in same function as well